Components/buttons/button.css

```diff
@@ -1,170 +1,189 @@
 .btn {
-    width: max-content;
+    /* Size properties */
     display: inline-block;
-    padding: var(--space-0-5) var(--space-0-75);
+    vertical-align: middle;
+    width: max-content;
     min-width: 80px;
```

Comment:
rems maye a better choice here. Also will be consistent.

Reply:
Hey, so var(--space-0-5) is 0.5rem here. 
Or are you referring to the width?

Reply:
I meant that for min-width property.

Reply:
Okay i shall change it to rems. Thank you.